vm/src/stdlib/socket.rs

```diff
@@ -449,6 +449,28 @@ fn socket_htonl(host: u32, vm: &VirtualMachine) -> PyResult {
     Ok(vm.new_int(host.to_be()))
 }
 
+fn socket_htons(host: u16, vm: &VirtualMachine) -> PyResult {
```

Comment:
You can have all these functions just return `u16` or `u32` rather than a PyResult -- `ctx.new_rustfunc` takes care of converting the return type.